client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
@himdel Hi Martin! Looking into this and it seems that you might be referring to `/api/automate_domains` to populate the tree? Or do you mean that the `resource_actions` of a service dialog needs to be passed back in the tree format?